Integration Tests BasicIT is failing with message about RocksdbMavenPomCache

I currenlty facing the issue related to the integration test `BasicIT`.

```
[INFO] Running org.openrewrite.maven.BasicIT
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 15.833 s <<< FAILURE! - in org.openrewrite.maven.BasicIT
[ERROR] null_check_profile_activation{MavenExecutionResult}  Time elapsed: 5.267 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting empty but was: ["Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache"]
	at org.openrewrite.maven.BasicIT.null_check_profile_activation(BasicIT.java:36)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   BasicIT.null_check_profile_activation:36 
Expecting empty but was: ["Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache"]
[INFO] 
[ERROR] Tests run: 24, Failures: 1, Errors: 0, Skipped: 1
[INFO] 
[INFO] 
```

The above is caused by the WARNING during the Maven build:
```
[INFO] Project [BasicIT#null_check_profile_activation] Resolving Poms...
[WARNING] Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache
[INFO] Project [BasicIT#null_check_profile_activation] Parsing Source Files
```

Testing environment:
```
Apache Maven 3.9.0 (9b58d2bad23a66be161c4664ef21ce219c2c8584)
Maven home: /Users/khm/tools/maven
Java version: 17.0.6, vendor: Eclipse Adoptium, runtime: /Users/khm/.sdkman/candidates/java/17.0.6-tem
Default locale: en_DE, platform encoding: UTF-8
OS name: "mac os x", version: "12.4", arch: "aarch64", family: "mac"
```

From my understanding the produced warning is related to the underlying OS (in my case MacOS)...

--- a/src/test/java/org/openrewrite/maven/BasicIT.java
+++ b/src/test/java/org/openrewrite/maven/BasicIT.java
@@ -7,7 +7,6 @@ import static com.soebes.itf.extension.assertj.MavenITAssertions.assertThat;
 
 @MavenJupiterExtension
 @MavenOption(MavenCLIOptions.NO_TRANSFER_PROGRESS)
-@SuppressWarnings("NewClassNamingConvention")
 class BasicIT {
 
     @MavenTest
@@ -33,6 +32,8 @@ class BasicIT {
                 .isSuccessful()
                 .out()
                 .warn()
+                // Ignore warning logged on Mac OS X; https://github.com/openrewrite/rewrite-maven-plugin/issues/506
+                .filteredOn(warn -> !"Unable to initialize RocksdbMavenPomCache, falling back to InMemoryMavenPomCache".equals(warn))
                 .isEmpty();
     }
 

